src/Mask.ts

```diff
@@ -512,12 +512,16 @@ export class Mask {
    * Get the coordinates of the points on the border of a shape defined in a mask.
    *
    * @param options - Get border points options.
-   * @returns Array of boder points.
+   * @returns Array of border points.
    */
   public getBorderPoints(options?: GetBorderPointsOptions): Point[] {
     return getBorderPoints(this, options);
   }
 
+  get ratio() {
+    return this.width / this.height;
+  }
```

Comment:
I think you can remove this for now.

If it's implemented is should be called `aspectRatio` and done for all types of images, not just masks.

You can keep it on ROIs.

Reply:
Can you point me to the definition of "aspect ratio" being a value between 0 and 1?

When searching I'm consistently finding that it's "the relationship between width and height"

Reply:
@stropitek I think the aspectRatio of the mask is calculated as part of  feretDiameter. It is minDiameter/maxDiameter. It is calculated in getFeret() function which is in a maskAnalysis folder

Reply:
This discussion is about the aspect ratio of Mask/Image, not regions of interest.

Reply:
Ah yes indeed. Didn't see there were changes in Mask as well. I though it was only on ROIs. I agree we can remove it for now.